pkgs/development/libraries/pe-parse/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "pe-parse";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "trailofbits";
+    repo = "pe-parse";
+    rev = "v${version}";
+    sha256 = "1jvfjaiwddczjlx4xdhpbgwvvpycab7ix35lwp3wfy44hs6qpjqv";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
```

Comment:
Could be a candidate for multiple outputs if you wish to reduce dependers' closure size
```nix
  outputs = [ "out" "dev" "bin" ];

  postInstall = ''
    moveToOutput bin $bin
  '';
```

Reply:
I had some trouble building another library this way. Maybe I'll leave the single output for now as it's not big and refactor that later if needed.